PaddleCV/image_classification/dist_train/dist_train.py

```diff
@@ -247,11 +261,12 @@ def train_parallel(args):
 
     strategy = fluid.ExecutionStrategy()
     strategy.num_threads = args.num_threads
+    strategy.num_iteration_per_drop_scope = 30
```

Comment:
The document has it already:
https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/fluid/pybind/pybind.cc#L1143

Reply:
I agree with @typhoonzero , a user with this repo may not be aware of https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/fluid/pybind/pybind.cc#L1143, and it will also be very time consuming to get the point.